src/FSharp.Data.GraphQL.Server/Planning.fs

```diff
@@ -273,7 +273,7 @@ and private planAbstraction (ctx:PlanningContext) (selectionSet: Selection list)
                 let infoMap, deferredFields', path' = Map.fold (foldPlan) (Map.empty, deferredFields, []) a
                 match field with
                 | Deferred -> fields, {Info = { innerData with Kind = ResolveAbstraction infoMap}; Path = path'; Kind = DeferredExecution} :: deferredFields'
-                | Streamed -> fields, {Info = { innerData with Kind = ResolveAbstraction infoMap}; Path = path'; Kind = StreamedExecution} :: deferredFields'
+                | Streamed -> fields, {Info = { innerData with Kind = ResolveAbstraction infoMap}; Path = path'; Kind = StreamedExecution OneByOne} :: deferredFields'
```

Comment:
By default, I am letting the default streamed execution here. The idea is to receive in schema metadata how the stream of a specific object should be done when asked to.